docs/source/yelpsoa_configs.rst

```diff
@@ -1069,6 +1069,35 @@ internally at Yelp.
      delay. Note that due to Envoy's current definition of *percentage* as an
      integer, this cannot be specified as a floating-point number.
 
+Error Alerting
+``````````````
+
+These keys provide optional overrides for the default alerting behaviour.
+
+ * ``monitoring``: Override default alerting behaviour. For example: ::
+
+      main:
+        monitoring:
+          team: frontend
+          slack_channel: "notifications"
+          jira_project: "FRONTEND"
+          ticket: false
+          page: false
+          page_nonprod: true
+          error_threshold_ration: 0.02
+          minimum_error_rps: 10
+    - ``team``: Override the default team for alerting.
+    - ``slack_channel``: Error alerts notify the first channel in the monitoring.yaml slack_channels list.
+      Use this key if you prefer a different channel.
+    - ``project``: Override the default JIRA project for alerting.
+    - ``ticket``: Override the default ticketing behaviour. Defaults to **true**.
```

Comment:
this should also be similar to the next line right?

Reply:
should it? There is no `ticket` in the monitoring.yaml schema

Reply:
ooo okokok gotcha

Reply:
should `ticket` default to true or false?  If we're defaulting `page` to true, it can probably be false?

Reply:
coming back to this, the schema is not consistent with what the docs say and there are many examples of ticket in monitoring.yaml so you are correct and i will update

Reply:
gotcha, thank you for checking!!